packages/cli-kit/src/plugins.ts

```diff
@@ -45,7 +45,7 @@ interface HookReturnsPerPlugin extends HookReturnPerTunnelPlugin {
   }
   [hookName: string]: {
     options: {[key: string]: unknown}
-    pluginReturns: {[key: string]: JsonMap | Result<JsonMap, Error>}
+    pluginReturns: {[key: string]: unknown}
```

Comment:
@shauns thoughts on this specifically? We couldn't find another way to allow plugins to return anything.